src/NumericalAlgorithms/LinearOperators/MeanValue.hpp

```diff
@@ -46,11 +49,42 @@ double mean_value(const DataVector& f, const Mesh<Dim>& mesh) noexcept {
  *
  * \returns the mean value of `f` on the boundary of the manifold
  *
- * \param f the grid function of which to find the mean.
- * \param mesh the Mesh of the manifold on which f is located.
- * \param d the dimension which is sliced away to get the boundary.
- * \param side whether it is the lower or upper boundary in the d-th dimension.
+ * - `f` the grid function of which to find the mean.
+ * - `mesh` the Mesh of the manifold on which f is located.
+ * - `d` the dimension which is sliced away to get the boundary.
+ * - `side` whether it is the lower or upper boundary in the d-th dimension.
+ * - `boundary_buffer` is a pointer to a DataVector of size
+ *   `mesh.slice_away(d).number_of_grid_points()` used as a temporary buffer
+ *   when slicing the data to the boundary. In the 1D case no buffer is required
+ *   and a `nullptr` may be passed.
```

Comment:
As the specialization already exists, is it more elegant to include a default for the unused variable?

Reply:
You can't default arguments that aren't the last ones (as in, you could default the last 5, but not the first 1)